core/src/main/java/org/pentaho/di/core/database/AzureSqlDataBaseMeta.java

```diff
@@ -0,0 +1,113 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2021 by Hitachi Vantara : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+package org.pentaho.di.core.database;
+
+import org.pentaho.di.core.exception.KettleDatabaseException;
+import org.pentaho.di.core.row.ValueMetaInterface;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * @author patitapaban19
+ *
+ */
+public class AzureSqlDataBaseMeta extends MSSQLServerDatabaseMeta {
```

Comment:
if your going to use AzureSQLDb,
Dont you have to change the name of this class too ?